app/grandchallenge/components/migrations/0022_alter_componentinterface_example_value.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.14 on 2024-08-14 08:06
```

Comment:
PRs should only introduce one migration per app. Delete both of these and regenrate them.

Reply:
Done